universal-application-tool-0.0.1/app/repository/ApplicantRepository.java

```diff
@@ -41,4 +41,8 @@ public ApplicantRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exe
         },
         executionContext);
   }
+
+  public Optional<Applicant> lookupApplicantSync(long id) {
+    return ebeanServer.find(Applicant.class).setId(id).findOneOrEmpty();
```

Comment:
suggestion (non-blocking) and praise: This is really clean, would you mind updating `lookupApplicant` (async) to match?

Reply:
Oh yeah - good catch! Done.

Reply:
Made the same update to ProgramRepository and QuestionRepository as well.